src/public.js

```diff
@@ -330,6 +331,7 @@ Grid.defaults = {
         iconRefresh: "glyphicon-refresh",
         iconSearch: "glyphicon-search",
         iconUp: "glyphicon-chevron-up",
+        iconReset: "glyphicon-flash",
```

Comment:
not sure about this one, doesn't speak clear/refresh/reset to me

Reply:
hi! hope on button title here )
![image](https://user-images.githubusercontent.com/36099472/186601981-3d9a32d6-55a7-428e-981b-942dab24d11a.png)
didn’t find a more suitable icon to be honest (so as not to be confused with the refresh button). floppy-remove may be? (like it even less)
![image](https://user-images.githubusercontent.com/36099472/186602443-9b7ffa2d-a718-4d5e-9391-bdc338a4485e.png)

Reply:
or glyphicon-random ?

Reply:
done)